components/TodoList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TodoListItem from './TodoListItem';
+
+function TodoList({todoList}){
+    return (
+        <ul>
+          {todoList.map((todo) => {
+            return (
+              <TodoListItem key = {todo.id} todo = {todo} />
```

Comment:
Do we want to be consistent on spacing between props and the equal sign and their value?